### PR TITLE
[5][cli] maintenance database update db structure

### PR DESCRIPTION
**Pull request by alikon**
Pull Request for Issue #30477

### Summary of Changes

added cli command like CMS "Maintenance: Database - Update Structure"

### Testing Instructions

from cli run
`php cli/joomla.php maintenance:database`
or
`php cli/joomla.php maintenance:database --fix`

### Actual result BEFORE applying this Pull Request
N/A

### Expected result AFTER applying this Pull Request
same as  "Maintenance: Database - Update Structure" from CMS

image

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [ ] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [ ] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by dautrich** [comment]
I've tested this PR locally with XAMPP (PHP 8.2.12).

On J5.1_Alpha2 the command doesn't exist:
Bildschirminhalt erfassen-5

**Comment by dautrich** [comment]
I have tested this item :white_check_mark: successfully on 163552fcadcb5d5cb9c29fbc431aead16a8b649d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42568">issues.joomla.org/tracker/joomla-cms/42568</a>.</sub>

File: libraries/src/Console/MaintenanceDatabaseCommand.php [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+     * Stores the Input Object
+     *
+     * @var    InputInterface
+     * @since  __DEPLOY_VERSION__
+     */
+    private $cliInput;
+
+    /**
+     * SymfonyStyle Object
+     * @var SymfonyStyle
```
Comment (Quy) [inline libraries/src/Console/MaintenanceDatabaseCommand.php]:
```suggestion
     * SymfonyStyle Object
     *
     * @var SymfonyStyle
```

File: libraries/src/Console/MaintenanceDatabaseCommand.php [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+            if ($item['errorsCount'] > 0) {
+                $errorCount = true;
+                if ($this->cliInput->getOption('fix')) {
+                    $model->fix([$item['extension']->extension_id]);
+                }
+            }
+        }
+
+        $this->ioStyle->table(
+            ['id', 'Name', 'Database', 'Manifest', 'note'],
```
Comment (Quy) [inline libraries/src/Console/MaintenanceDatabaseCommand.php]: Use language strings.

Comment (Hackwar) [reply libraries/src/Console/MaintenanceDatabaseCommand.php]: No, on CLI we are only using english.

**Comment by nielsnuebel** [comment]
I have tested this item ✅ successfully

**Comment by nielsnuebel** [comment]
I have tested this item :white_check_mark: successfully on 02534fa57b76cdf5fa5f5be1a13c644c3da5a0ac<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42568">issues.joomla.org/tracker/joomla-cms/42568</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 02534fa57b76cdf5fa5f5be1a13c644c3da5a0ac<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42568">issues.joomla.org/tracker/joomla-cms/42568</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42568">issues.joomla.org/tracker/joomla-cms/42568</a>.</sub>

**Comment by bembelimen** [comment]
Thanks

**Pull request merged**